ayushma/models/users.py

```diff
@@ -5,20 +5,38 @@
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
+def reverse_choices(choices):
+    output = {}
+    for choice in choices:
+        output[choice[0]] = choice[1]
+    return output
 
 class UserManager(BaseUserManager):
     pass
 
 
 class User(AbstractUser):
-    first_name = None
-    last_name = None
+    first_name = models.CharField(max_length=32, blank=False, null=False)
+    last_name = models.CharField(max_length=32, blank=True, null=True)
 
```

Comment:
```suggestion
    first_name = None
    last_name = None
```
The `full_name` field does the work for us, we don't need these two!